templates/localstack_aws_credentials.j2

```diff
@@ -0,0 +1,3 @@
+[default]
```

Comment:
Random Q: Would it help to use a non-default profile? (The store has to specify which profile in that case.)

Reply:
@qqmyers these are broken out by service account, and really only for testing, so I think I'll leave them as-is for now? can always add in profiles later, unless you think I'm making a mistake by not including them now?

Reply:
It's fine - not hard to add later